### PR TITLE
0.9.1

### PR DESCRIPTION
**Pull request by BartoszLitwiniuk**

### CODE REVIEW

**Comment by konraddysput** [comment]
@BartoszLitwiniuk if you don't mind - can we ask @rqbacktrace to review this code?

File: build.gradle [hunk 1]
```diff
@@ -6,21 +6,20 @@ apply from: 'publish.gradle'
...
-version = '0.9.0'
+version = '0.9.1'
 
 repositories {
     jcenter()
 }
 
 dependencies {
-    implementation 'com.github.backtrace-labs.backtrace-java:backtrace-java:0.9.1'
-    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.9'
```
Comment (rqbacktrace) [inline build.gradle:17]: We don't need this dependency anymore?

**Review by rqbacktrace — APPROVED** [review]
Looks good to me!

File: build.gradle (hunk 1 above)
Comment (rqbacktrace) [reply build.gradle:17]: @BartoszLitwiniuk tested it builds and is able to be imported without this dependency. Let's remove it!

Comment (BartoszLitwiniuk) [reply build.gradle:17]: I was testing everything on independent project without this dependency and everything was working. I will also test it from staging to make sure that we don't break anything.

**Pull request merged**